src/niscope/examples/niscope_fetch_info.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python
+
+import argparse
+import niscope
+import numpy
+import pprint
+import sys
+
+pp = pprint.PrettyPrinter(indent=4, width=80)
+
+
+def example(resource_name, channels, options, length, voltage):
+    # This method of fetching is more complicated, but much faster than fetch().
```

Comment:
```suggestion
    # fetch_into() allows you to pre-allocate and re-use the destination of the fetched waveforms, which can result in better performance at the expense of the usability of fetch().
```

Reply:
I see you use preallocate bellow (no hyphen). Not sure which is right just be consistent.